.github/workflows/build_and_deploy_hold.yml

```diff
@@ -0,0 +1,94 @@
+name: Build and deploy with a hold state
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  call_build:
+    uses: ./.github/workflows/build.yml
+    with:
+      build_script: build-locales
+
+  call_unit_test:
+    uses: ./.github/workflows/unit_test.yml
+    needs: call_build
+
+  call_misc_tests:
+    uses: ./.github/workflows/miscellaneous_tests.yml
+
+  call_acceptance:
+    uses: ./.github/workflows/acceptance.yml
+    needs: call_build
+    secrets:
+      BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
+      BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
+
+  call_format_branch_name:
+    uses: ./.github/workflows/format_branch_name.yml
+
+  call_extract_versions:
```

Comment:
I was using the `call` convention to indicate that we were using the callable workflows feature. Since this is a standard job, could we instead call this `extract_versions`?